cyhy/mailer/__init__.py

```diff
@@ -1,3 +1,3 @@
 """This package contains the cyhy-mailer code."""
 
-__version__ = "1.7.0"
+__version__ = "1.7.1"
```

Comment:
There is no reason to change the version.

```suggestion
__version__ = "1.7.0"
```

Reply:
If this is accepted, please just drop the version bump commit.

Reply:
I changed the version so that I wouldn't have to re-create the 1.7.0 tag and release, which currently refers to `cisagov/cyhy-mailer:1.6.0`.  I'm fine doing that here if you guys think we should, but it seemed cleaner to me to just move to 1.7.1.

Reply:
A version bump indicates a change in the package and I don't think adjusting a Docker composition that is stored here (I'm working on moving it out) should impact the version of the package.

Reply:
I think you should recreate the tag and release.

Reply:
I cleaned this PR up and force-pushed https://github.com/cisagov/cyhy-mailer/pull/108/commits/25702fa363a046f51cd69982234fbdee3f2ec3e0.  Please let me know if you have any other issues here.